src/request/import/ImportWords.js

```diff
@@ -22,7 +22,7 @@
 
 class ImportWords {
     /**
-     * @param {Parsed<KeyguardRequest.ImportRequest>} request
+     * @param {Parsed<KeyguardRequest.ImportRequest | KeyguardRequest.ResetPasswordRequest>} request
```

Comment:
Might be that this is something different than `Parsed<KeyguardRequest.ImportRequest> | Parsed<KeyguardRequest.ResetPasswordRequest>` but I'm not sure.

Reply:
I don't think it is, at least not in this scenario.
```
request: Transform<KeyguardRequest.ImportRequest, ...> | Transform<KeyguardRequest.ResetPasswordRequest, ...>
```
is the resulting type.